cmd/generator/main.go

```diff
@@ -177,9 +182,16 @@ func getReleasePlugin(ctx context.Context, client *github.Client, repositoryName
 	logger.Debugf("found latest release %s", releaseName)
 
 	downloadURL := ""
+	var updatedAt time.Time
 	for _, releaseAsset := range latestRelease.Assets {
 		if strings.HasSuffix(releaseAsset.GetName(), ".tar.gz") {
 			downloadURL = releaseAsset.GetBrowserDownloadURL()
+			timestampUpdatedAt := releaseAsset.GetUpdatedAt()
+			if timestampUpdatedAt.IsZero() {
+				timestampUpdatedAt = releaseAsset.GetCreatedAt()
+			}
+
+			updatedAt = timestampUpdatedAt.In(timestampUpdatedAt.Location())
```

Comment:
I'm wondering if we not just use `UTC` all the time (haha).

Reply:
I originally wrote this was the closest thing I could fine to a `Clone` -- just copying the timezone from the incoming timestamp. I'm assuming UTC at the source, but I suppose it doesn't hurt to enforce.